map_model/src/edits/mod.rs

```diff
@@ -548,7 +541,11 @@ impl Map {
             lanes_ltr: r
```

Comment:
Is the idea that eventually this will move farther up the call stack, so that Road.lanes becomes a vec of LaneSpecs?

```diff
- pub lanes_ltr: Vec<(LaneID, Direction, LaneType)>,
+ pub lanes_ltr: Vec<(LaneID, LaneSpec)>,
```

Reply:
That would simplify some uses of it. I'll toss this in the next batch of map format changes.